setup_env.bat

```diff
@@ -24,6 +24,7 @@ call git submodule update
 
 call android.bat update project -p libs/android-support-appcompat-v7-exploded-aar --target android-23
 call android.bat update project -p libs/android-support-design-exploded-aar --target android-23
+call android.bat update project -p libs/com-getbase-floatingactionbutton-1-10-1-exploded-aar --target android-23
```

Comment:
The target number here should coincide with the value of targetSdk in libs/com-getbase-floatingactionbutton-1-10-1-exploded-aar/AndroidManifest.xml ; would be awesome if 23 works, though if 22 was the original in the Manifest, problems could arise.